master/internal/kubernetes/request_queue.go

```diff
@@ -190,7 +207,18 @@ func (r *requestQueue) receiveDeleteKubernetesResources(
 	r.queue = append(r.queue, &queuedResourceRequest{deleteResources: &msg})
 }
 
-func (r *requestQueue) receiveWorkerAvailable(ctx *actor.Context) {
+func (r *requestQueue) receiveWorkerAvailable(ctx *actor.Context, msg workerAvailable) {
+	if msg.resourceHandler != nil {
```

Comment:
blocking: `msg.resourceHandler == nil` seems impossible. I don't think we should handle it.